src/app/CommandInputField.tsx

```diff
@@ -71,6 +74,11 @@ export const CommandInputField = observer((props: { controller: BottomPanelContr
       const selected = controller.selected;
       const autoCompletionValues = controller.autoCompletionValues;
 
+      if (selected === null && autoCompletionValues.length <= 0 && lastCmd.length > 0) {
+        lastCmdIndex.set(lastCmdIndex.get() - 1 === -1 ? 0 : lastCmdIndex.get() - 1);
+        input.value = lastCmd[lastCmdIndex.get()];
```

Comment:
Also, set the carpet position to the end of the command line after using the hotkey.